erpnext/crm/doctype/email_campaign/email_campaign.py

```
@@ -77,58 +77,128 @@ def update_status(self):
+			email_template=email_template.name,
+		)
+
+		frappe.sendmail(
+			recipients=recipient_list,
+			subject=subject,
+			content=content,
+			sender=sender,
+			communication=comm["name"],
+			queue_separately=True,
+		)
+	except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)